HISTORY.rst

```diff
@@ -1,6 +1,28 @@
 Release History
 ---------------
 
+0.8.0 (2019-10-??)
+++++++++++++++++++
+Major release with several breaking changes.
+
+* API change: Dropped support for Python 2
+* API change: Gathering of git information gathering is now enabled by default #595
+* API change: Switched constructor from Observer.create(...) to Observer(...) for all observers.
+* API change: Changed the interface for collecting custom host-information #569
+* API change: Changed interface for defining CLI options. #572
+* API change: Started using pathlib.Path instead of os.path in many places
```

Comment:
I'm not sure it's an API change. We stayed backward compatible and still support passing strings as paths. At least it would be a bug on our side if the user got an exception because he/she passed a string as input somewhere.

Reply:
That's true. Shall we list it as internal? Or skip it?

Reply:
list as internal I believe.